dvc/tree/s3.py

```diff
@@ -68,16 +69,23 @@ def s3(self):
             "profile_name": self.profile,
             "region_name": self.region,
         }
+        config = None
 
         if self.access_key_id:
             session_opts["aws_access_key_id"] = self.access_key_id
         if self.secret_access_key:
             session_opts["aws_secret_access_key"] = self.secret_access_key
+        if self.session_token:
+            session_opts["aws_session_token"] = self.session_token
+            config = boto3.session.Config(signature_version="s3v4")
```

Comment:
out of curiosity - is it required? is it always `s3v4`?

Reply:
I've just checked the docs, yes it is. Thanks for catching it!